include/mqtt/property.hpp

```diff
@@ -11,6 +11,8 @@
 #include <vector>
 #include <memory>
 #include <algorithm>
+#include <ostream>
```

Comment:
ostream is very expensive to include.

Recommend using iosfwd instead, since you only use this with template functions.

Reply:
Thanks! Fixed.